tests/test_amity.py

```diff
@@ -1,146 +1,299 @@
 import unittest
-from unittest.mock import patch
-
 from app.amity import *
 
 
 class AmityTestCase(unittest.TestCase):
+
     def setUp(self):
         self.amity = Amity()
 
     def test_adds_person(self):
+        """Test it adds a person to Amity"""
         person_count_before = len(self.amity.all_people)
-        self.amity.add_person("Ivan", "fellow", "y")
-        person_count_after = len(self.amity.all_people)
-        self.assertEqual(person_count_after, person_count_before + 1)
-
-    def test_add_person_duplicate_name(self):
-        self.amity.add_person("Chris", "staff")
-        people_names = [people.person_name for people in self.amity.all_people]
-        self.assertIn("Chris", people_names)
-        self.assertEqual(self.amity.add_person("Chris", "staff"),
-                         "sorry, this user already exists.please choose another name")
-
-    def test_add_person_no_room(self):
-        self.assertEqual(self.amity.add_person("Sam", "Fellow", "Y"), "sorry, all rooms are full at this time.")
-
-    def test_add_person_not_allow_more_than_capacity_office_allocation(self):
-        people_list = {"Sam": "Fellow", "Gideon": "Fellow", "Charles": "Fellow", "Rogers": "Staff", "Mahad": "Fellow",
-                       "Percila": "Staff", "Brian": "Fellow"}
-        self.amity.create_room("office", "Mordor")
-        for name in people_list:
-            assigning_office = self.amity.add_person(name, people_list[name])
+        self.amity.add_person("cn03", "Ivan", "Kipyegon", "fellow", "y")
+        self.assertEqual(len(self.amity.all_people), person_count_before + 1)
+
+    def test_add_person_duplicate_employee_number(self):
```

Comment:
test message should be more descriptive i.e `test_error_adding_person_with_duplicate_employee_number`